### PR TITLE
fix path resolve for usb-native libs

### PR DESCRIPTION
**Pull request by chagong**
fix path resolve for usb-native libs

### CODE REVIEW

File: lib/native_loader.js [hunk 1]
```diff
@@ -17,7 +17,8 @@ function loadLibrary(libraryName, destPath) {
     console.log('using default', destPath);
   } catch (e) {
     // Copy pre-compiled node binary to dest path
-    const parentFolder = path.join(__dirname, "native");
+    const libFolder = typeof __webpack_require__ === "function" ? __dirname + "/../node_modules/node-usb-native/lib/" : __dirname;
```
Comment (dilin-MS) [inline lib/native_loader.js:20]: Can we put webpack-related logic into utility for better re-usage?

Comment (chagong) [reply lib/native_loader.js]: Done.

File: lib/utils.js [hunk 2]
```diff
@@ -8,4 +8,10 @@ function getNodeFilePath(relativePath) {
     return path.join(__dirname, destPath);
 }
 
-exports.getNodeFilePath = getNodeFilePath;
\ No newline at end of file
+function getNodeLibFolder() {
```
Comment (dilin-MS) [inline lib/utils.js:11]: These two methods `getNodeFilePath()` and `getNodeLibFolder()` ooks very much alike. Could we compact them into one method which handles situation when getting path (webpack or not)?

Comment (chagong) [reply lib/utils.js:11]: I think it's possible, let me check.

Comment (chagong) [reply lib/utils.js]: It's done. Good comments.

File: lib/native_loader.js [hunk 3]
```diff
@@ -17,7 +17,8 @@ function loadLibrary(libraryName, destPath) {
     console.log('using default', destPath);
   } catch (e) {
     // Copy pre-compiled node binary to dest path
-    const parentFolder = path.join(__dirname, "native");
+    const relativePath = "node-usb-native/lib/native/";
+    const parentFolder = require("./utils").getNodeFilePath(relativePath);
```
Comment (dilin-MS) [inline lib/native_loader.js:21]: The function name may need refreshing.

Comment (chagong) [reply lib/native_loader.js]: done.

**Pull request merged**